Kernel/FileSystem/RAMFS/Inode.cpp

```diff
@@ -127,7 +127,7 @@ ErrorOr<size_t> RAMFSInode::read_bytes_from_content_space(size_t offset, size_t
 {
     VERIFY(m_inode_lock.is_locked());
     VERIFY(m_metadata.size >= 0);
-    if (static_cast<size_t>(m_metadata.size) < offset)
+    if (offset >= static_cast<size_t>(m_metadata.size))
         return 0;
```

Comment:
What exactly is this supposed to fix? :thinking:

Reply:
Oops, I got myself confused here. After this change, we now reject attempts to read if the offset is exactly after the end of the file.